assemblies/assembly_kogito-creating-running/main.adoc

```diff
@@ -0,0 +1,48 @@
+[id='assembly_kogito-creating-running']
+
+include::_artifacts/document-attributes.adoc[]
+
+= Creating and running your first application with {PRODUCT} services
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: kogito-creating-running
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a developer of business processes and rules, you can use {PRODUCT} business automation to build  cloud-native applications that adapt to your business domain and tooling.
+
+[IMPORTANT]
+====
+{PRODUCT} is currently for Developer Preview only. Developer Preview releases contain features that might not be fully tested and are not in production-ready state. Customers are advised to avoid running the software for production or business-critical workloads. Customers are not able to open support cases against the Developer Preview releases. Red Hat does not commit to fixing any reported issues and the provided features might not be available in future releases.
+====
+
+.Prerequisites
+////
+// @comment: Removed for now but retaining for use in next docs. (Stetson, 18 Feb 2020)
+* One of the following IDEs is installed:
+** VSCode (recommended) with the {PRODUCT} https://github.com/kiegroup/kogito-tooling/releases[VSCode Extension] installed for BPMN 2.0 and DMN modeling support
+**  Eclipse with BPMN2 Modeler plugin with jBPM Runtime Extension for BPMN 2.0 modeling support
```

Comment:
@sterobin I would suggest removing all mentions of Eclipse plugin on our docs. Our tooling story is all based on VSCode.

Reply:
The same also applies for our website.

Reply:
@ederign, I've just redone this section entirely to be about "Enagling the VSCode extension" instead, since that's all that's left. This promotes the extension and clarifies things. Here's the new section in the PR: https://github.com/sterobin/kie-docs/pull/32/files#diff-954fa17f16cfba12dc4eb802afc46508  And here's the new section in the rendered doc (same for both comm and enterprise): http://file.rdu.redhat.com/~sterobin/BXMSDOC-5383/#proc_kogito-vscode-extension  Refresh the preview page.

This work for you?